main/wallet.js

```diff
@@ -21,7 +21,7 @@ const walletStore = new Store({
 
 const backend = new WalletBackend({
   async onTransactionUpdate () {
-    walletStore.set('transactions', backend.transactions)
+    walletStore.set('transactions_0x', backend.transactions)
```

Comment:
Can we use the same seed phrase? I think the seed phrase format standardised and it's possible to generate different addresses (BTC, ETH, FIL) from the same seed.

See:
- [BIP-0039 Mnemonic code for generating deterministic keys](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki)
- [BIP-0032 Hierarchical Deterministic Wallets](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)
- https://github.com/satoshilabs/slips/blob/master/slip-0010.md
- https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki
- https://github.com/bitcoin/bips/blob/master/bip-0049.mediawiki
- https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki

I am concerned about the upgrade path and losing access to funds stored in existing Station wallets.

Reply:
Ouch, I got confused. This is not the seed phrase but the local storage.

Please ignore my comment above.